Update squeezebox.markdown for https support (#27939)
home-assistant/home-assistant.github.io: source/_integrations/squeezebox.markdown

{% include integrations/config_flow.md %}

<div class='note'>

> This platform uses the web interface of the Logitech Media Server to send commands. The default port of the web interface is 9000. It is the same port that you use to access the LMS through your web browser.

Reviewer: ```suggestion
This platform uses the web interface of the Logitech Media Server (LMS) to send commands. The default port of the web interface is 9000. It is the same port that you use to access the LMS through your web browser.
```

</div>

<div class='note'>

> The integration now supports Logitech Media Servers behind an https reverse proxy. Please note that Logitech Media Server only natively supports http traffic, so do not select the `https` option unless you have configured such a reverse proxy. Remember you will likely need to update the port from the default as well.

Reviewer: ```suggestion
The integration now supports Logitech Media Servers behind an HTTPS reverse proxy. Please note that Logitech Media Server natively only supports HTTP traffic. Unless you have configured a reverse proxy, do not select the `https` option. If you have configured a reverse proxy, remember to update the port number.
```
